test/stateful_dataloader/test_dataloader.py

```diff
@@ -1638,6 +1638,7 @@ def test_multiprocessing_contexts(self):
                     list(self._get_data_loader(ds_cls(counting_ds_n), multiprocessing_context=ctx, **dl_common_args)),
                 )
 
+    @unittest.skipIf(IS_MACOS, "Not working on macos")
```

Comment:
See `test_multiprocessing_contexts()` above on line 1614, similar non-datapipe test is skipped for macOS so going to follow that here too

Reply:
An alternative for this test to pass is to skip `ctx == "fork"`